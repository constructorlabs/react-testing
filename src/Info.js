import React from 'react';

class Info extends React.Component {
  render(){
    return (
      <div>
        <h1>See info below</h1>
        <p>
          Incredibly useful information.
        </p>
      </div>
    );
  }
}

export default Info;
