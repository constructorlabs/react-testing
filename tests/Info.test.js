import React from 'react';
import Info from '../src/Info';
import renderer from 'react-test-renderer';

describe('Info', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Info />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
