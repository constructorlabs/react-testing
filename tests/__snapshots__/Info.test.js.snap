// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info matches the snapshot 1`] = `
<div>
  <h1>
    See info below
  </h1>
  <p>
    Incredibly useful information.
  </p>
</div>
`;
